docs/code_contribution_guidelines.md

```diff
@@ -375,6 +401,34 @@ Notice how the `json_name` field option corresponds with the name of the field
 itself, and uses a `snake_case` style of name formatting. All added or modified
 `proto` fields should adhere to the format above.
 
+<a name="ExtraGoFmtStyle" />
+
+#### 4.8. Additional Style Constraints On Top of `gofmt`
+
+Before a PR is submitted, the proposer should ensure that the file passes the
+set of linting scripts run by `make lint`. These include `gofmt`. In addition
+to `gofmt` we've opted to enforce the following style guidelines.
+
+   * ALL columns (on a best effort basis) should be wrapped to 80 line columns.
+     Editors should be set to treat a tab as 4 spaces.
+   * When wrapping a line that contains a function call as the unwrapped line
+     exceeds the column limit, the close paren should be placed on tis own
+     line. Additionally, all arguments should begin in a new line after the
+     open paren.
+
+     **WRONG**
+     ```go
+     value, err := bar(a
+        a, b, c)
+     ```
+
+     **RIGHT**
+     ```go
+     value, err := bar(
+        a, a, b, c,
+     )
+     ```
+
```

Comment:
Do we even do this with only a single arg that doesn't fit the 1st line?

```
value, err := bar(
    inputValue,
)
```

Reply:
If it doesn't fit the line, I think we do.

Reply:
Should we add a note that this doesn't apply to log messages?

Reply:
Done!